extensions/wikia/UserProfilePageV3/UserTagsStrategy.class.php

```diff
@@ -0,0 +1,224 @@
+<?php
+use \Wikia\DependencyInjection\Injector;
+use \Wikia\Service\User\Permissions\PermissionsService;
+
+/**
+ * Class for handling user "tags" logic in user masthead
+ * @class UserTagsStrategy
+ */
+class UserTagsStrategy extends WikiaObject {
+
+	/** @var int MAX_USER_TAGS Positive even integer indicating at most how many user tags can be shown */
+	const MAX_USER_TAGS = 2;
+
+	/** @var array GLOBAL_GROUPS_RANK Global groups that should be shown in masthead, in order of importance  */
+	const GLOBAL_GROUPS_RANK = [
+		'staff',
+		'helper',
+		'vstf',
+		'voldev',
+		'wikiastars',
+		'vanguard',
+		'council',
+	];
+
+	/** @var array LOCAL_GROUPS_RANK Local groups that should be shown in masthead, in order of importance  */
+	const LOCAL_GROUPS_RANK = [
+		'bureaucrat',
+		'sysop',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "sysop" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)